typescript/src/client/evm/evm.ts

```diff
@@ -295,6 +304,54 @@ export class EvmClient implements EvmClientInterface {
     }
   }
 
+  /**
+   * Gets a quote for a swap between two tokens on an EVM network.
+   *
+   * @param {GetSwapQuoteOptions} options - The options for getting a swap quote.
```

Comment:
Will want to add the individual fields of GetSwapQuoteOptions as we do with the other methods